src/shop-client.js

```diff
@@ -142,7 +142,7 @@ const ShopClient = CoreObject.extend({
    * ```
    *
    * @method fetchQuery
-   * @public
+   * @private
```

Comment:
Good question. Essentially, they're the methods that provide the behaviour leveraged by higher-level functions. The main concern is that they introduce a large amount of complexity to the documentation, because it introduces two ways of doing the same thing. They're not really private, _or_ protected, they just provide the behaviour behind the API we want to expose. Not documenting these might be a good approach as well. Hmmm... 🤔